pimcore/lib/Pimcore/Console/Command/ThumbnailsImageCommand.php

```diff
@@ -98,7 +98,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 }
 
                 if ($input->getOption("system")) {
-                    $thumbnail = Asset\Image\Thumbnail\Config::getPreviewConfig();
+                    $thumbnail = \Pimcore::getDiContainer()->get('Pimcore\Model\Asset\Image\Thumbnail\Config');
+                    $thumbnail = $thumbnail::getPreviewConfig();
```

Comment:
The [`getPreviewConfig()` method is static](https://github.com/pimcore/pimcore/blob/5036920fca970ab124914d55aed9180cf50541dd/pimcore/models/Asset/Video/Thumbnail/Config.php#L101). Using the container means getting back an *instance* of the class: it's not useful in that case since static methods are not executed on instances.